content/community/teams.md

```diff
@@ -5,53 +5,55 @@ next: /community/rules/
 
 # Teams and Organization
 
-_This page is a stub._
-
 ## Administrators
 
 Administrators are people who created, maintain, and are responsible for Codewars overall. They handle the most critical issues, like:
-- severe cases of violations of rules,
-- user accounts,
-- bugs,
-- ...?
+- Development and maintenance of Codewars platform,
+- Fixing platform bugs related to Codewars functions, user interface, code runner,
+- Managing user accounts,
+- Handling severe cases of violations of rules.
 
-Cuttently, users with status of Administrator are:
+Cuttently, active users with status of Administrator are:
 
-- [kazk](link to Codawars profile?)
-- ... others?
+- [kazk](https://www.codewars.com/users/kazk)
 
 They can be recognized by symbol of a crown (👑) displayed by their user name.
 
 ## Moderators
 
-Moderators are small team of users selected by Administrators to support them in handling some tasks related to community and Codewars content. Moderators have tools and privileges which help with maintenance of content and with handling of some community issues. Moderators can:
-- moderate discourses,
-- validate or flag solutions,
-- identify cheaters and abusers,
-- revoke privileges,
-- maintain documentation.
+Moderators are small team of users selected by Administrators to support them in handling some tasks related to community and Codewars content. Moderators have tools and privileges which help with maintenance of content and with handling of some community issues. Moderators have access to tools which allow them to:
+- Moderate discourses,
+- Manage submitted solutions: re-test them, flag as cheats, etc.
+- Identify cheaters, plagiarists, and users who do not play the game fair,
+- Handle abusers and revoke their privileges,
+- Maintain documentation.
 
 Moderators also serve as intermediaries (word?) between Codewars users and Administrators, or vice versa. They can pass anouncements from maintainers, or support users in communication when some issue needs to be brought to attention of Administrators.
 
-But still, Moderators are just regular users, and as such, they're subject to all Codewars rules. In general, they **cannot**:
-- see solutions to kata which they have not unlocked,
-- fix problems with kata they have no privileges to coauthor,
-- modify kata in a way which is not directly supported by Codewars: change its ranking, move it back to beta, retire or delete it,
-- ...?
+But still, Moderators are just regular users, and as such, they're subject to all Codewars rules. In particular, they **cannot**:
+- Fix problems with kata they have no privileges to coauthor,
+- Approve translations which cannot be normally approved,
+- Modify kata in a way which is not directly supported by Codewars: change its ranking, move it back to beta, retire or delete it.
```

Comment:
Do we need this section? Seems redundant to me because there's already a section about what mods can do.

Also,

> - Fix problems with kata they have no privileges to coauthor,

I'll allow this in the future.

> - Approve translations which cannot be normally approved,

I might allow this in the future if it makes sense. Maybe require more than one user to approve.

> - Modify kata in a way which is not directly supported by Codewars: change its ranking, move it back to beta, retire or delete it.

I might allow moving back to beta if it's within few hours and if multiple mods agree. This will allow reranking for new kata.

I might also allow manually retiring beta kata that's been stuck for a while and when it's unlikely to be ever approved.

Reply:
I included it to limit situations when mods get flooded with requests for handling situations they cannot handle.

It's kinda preemptive, so I will remove it now and add back if it turns out necessary ;)